pkg/project/git.go

```diff
@@ -60,35 +60,35 @@ func (g Git) Detect() (Result, bool, error) {
 	}
 
 	// Find for .git/config file
-	gitConfigFile, ok := findGitConfigFile(fp, ".git", "config")
+	gitConfigFile, ok := findFileOrDirectory(fp, ".git", "config")
 
 	if ok {
-		project := filepath.Base(filepath.Join(gitConfigFile, ".."))
+		project := filepath.Base(filepath.Join(gitConfigFile, "../.."))
```

Comment:
To be windows compatible, I guess this needs a small adjustment.
```
project := filepath.Base(filepath.Join(gitConfigFile, "..", ".."))
```

But in general, we're doing a little hack here, right? Because `findFileOrDirectory` will return something like `/path/to/wakatime-cli/.git/config` and then we're pretending `config` is a directory by adding `../..`. How about to strip the filename and continue with the `.git` path?

Reply:
> To be windows compatible, I guess this needs a small adjustment.

Tests on windows pass 🤔 

> But in general, we're doing a little hack here, right? 

Did you mean to change the returned value?

Reply:
> Did you mean to change the returned value?
What I meant w

Reply:
To make it a little bit more clean, we could use `filepath.Split` (https://golang.org/pkg/path/filepath/#Split), to split the filename and then work with `/path/to/wakatime-cli/.git` in the code below.

Reply:
I see. Makes more sense though 👍